src/server/conn_context.h

```diff
@@ -206,6 +203,8 @@ class ConnectionContext : public facade::ConnectionContext {
   std::string authed_username{"default"};
   uint32_t acl_categories{acl::ALL};
   std::vector<uint64_t> acl_commands;
+  // Skip ACL validation, used by internal commands and commands run on admin port
+  bool skip_acl_validation = false;
```

Comment:
This is particularly nice, because we create adhoc contexts in different scenarios. For example, during replication or squashing. 

Why we need this? Partly because of this PR, but also because by default a connection context is created with the user `default`. BUT, `DFLY` command requires `ADMIN` `constexpr uint32_t kDfly = ADMIN;` and it can be the case, that the DF admin removed `ADMIN` acl from `default`. Currently, all `ConnectionContexts` do no read the current value of `default` user from the registry but rather, they create a context with the user "default" BUT with all the credentials. What we want is to decouple the commands issued by users, and the ones issued internally by DF. This handles that and I can finally introduce (in another PR) the fix for the ConnectionContext being constructed with full ACL's (even if default, does not have them).